tests/resources/docker-compose.yml

```diff
@@ -2,15 +2,19 @@ version: '3'
 
 services:
   appwrite:
-    build: ../../
+    build:
+      context: ../../
+      args:
+        - TESTING=true
     restart: unless-stopped
     volumes:
     - appwrite-uploads:/storage/uploads:rw
     - appwrite-cache:/storage/cache:rw
+    - ./../../phpunit.xml:/usr/share/nginx/html/phpunit.xml
```

Comment:
Any reason why we need the tests inside the container?

Reply:
Because the environment outside docker is not prepared to run anything related to tests, so for example, all packages are installed inside the container itself so I decided to move the tests as well to avoid any repeated steps and so on